pkgs/development/libraries/qt-5/5.15/srcs.nix

```diff
@@ -3,7 +3,7 @@
 let
   version = "5.15.6";
   overrides = {
-    qtscript.version = "5.15.4";
+    qtscript.version = "5.15.6";
```

Comment:
This is equivalent to removing the override since it matches.

Reply:
Good catch, thanks!